packages/components/src/spacer/stories/index.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { Spacer } from '..';
+
+const controls = [
+	'margin',
+	'marginY',
+	'marginX',
+	'marginTop',
+	'marginBottom',
+	'marginLeft',
+	'marginRight',
+
+	'padding',
+	'paddingY',
+	'paddingX',
+	'paddingTop',
+	'paddingBottom',
+	'paddingLeft',
+	'paddingRight',
+].reduce(
+	( acc, prop ) => ( { ...acc, [ prop ]: { control: { type: 'number' } } } ),
```

Comment:
I just noticed [this discrepancy](https://github.com/WordPress/gutenberg/issues/41882) in the `SpaceInput` type description. We can keep these `number` controls for now, but we might end up changing them to `text` later. Not sure yet! Up for discussion.

Reply:
I feel its easier to test the story when I can use the arrow keys to change the spacing, but as the issue points out it would be great to use all supported values.

Happy to change it to `text` and get it correct.